akka-osgi/build.sbt

```diff
@@ -1,7 +1,7 @@
-import akka.{ AkkaBuild, Dependencies, Formatting, OSGi }
+import akka.{ AkkaBuild, Dependencies, Formatting, OSGi, Dist }
 
 AkkaBuild.defaultSettings
-AkkaBuild.dontPublishSettings
+Dist.includeInDist := false
```

Comment:
I'm not sure, but I don't osgi can be used without maven repo, so it has no value in the dist